frontend/www/js/omegaup/components/group/Identities.vue

```diff
@@ -110,7 +123,11 @@ export default class Identities extends Vue {
     }
 
     this.identities = [];
-    this.$emit('read-csv', { identities: this.identities, file: file });
+    this.$emit(
+      'read-csv',
+      { identities: this.identities, file: file },
+      this.humanReadable,
```

Comment:
```suggestion
      { identities: this.identities, file: file, humanReadable: this.humanReadable },
```

estamos intentando que todos los emit tengan a lo más un argumento.